binary_search/BinarySearch.java

```diff
@@ -0,0 +1,39 @@
+
+public class BinarySearch {
+
+    public static void main(String[] args) {
+        int[] arr = {1, 5, 35, 112, 258, 324},
+                searchArr = {1, 35, 112, 324, 67};
```

Comment:
Make array declaration as
```java
int[] arr = new int[] {1, 5, 35, 112, 258, 324};
```
Also each variable should have its own declaration statement.